src/lib/targeting.ts

```diff
@@ -0,0 +1,79 @@
+import { Tag } from '../components/ContributionsWrapper';
+
+// Content types to be considered for an Epic
+const ACCEPTED_TYPES = ['Article'];
+
+// SECTION / TAG BLACKLIST
+// Candidates to be abstracted into a config layer
+const SECTION_BLACKLIST: string[] = [
+    'football',
+    'money',
+    'education',
+    'games',
+    'teacher-network',
+    'careers',
+];
+
+// The types of tag our targeting checks support
+// Others could be 'Tone', 'Publication', 'Tracking', etc.
+type TagType = 'Keyword';
+
+interface TagList {
+    Keyword: string[];
+}
+
+const TAG_BLACKLIST: TagList = {
+    Keyword: ['guardian-masterclasses/guardian-masterclasses'],
+};
+
+// Determine if it's the right type of content to be considered for an Epic
+export const isEpicContent = ({ contentType }: { contentType: string }): boolean => {
+    return ACCEPTED_TYPES.includes(contentType);
+};
+
+// Determine if the content is suitable for an Epic
+export const isEpicSuitable = ({
+    shouldHideReaderRevenue,
+    isMinuteArticle,
+    isPaidContent,
+}: {
+    shouldHideReaderRevenue: boolean;
+    isMinuteArticle: boolean;
+    isPaidContent: boolean;
+}): boolean => {
+    const foundUnmetCondition = [shouldHideReaderRevenue, isMinuteArticle, isPaidContent].some(
+        condition => condition,
```

Comment:
It took me a couple of reads to make sure I understood this correctly. I wonder if this would be clearer:

```js
return !(shouldHideReaderRevenue || isMinuteArticle || isPaidContent);
```

What do you think? Maybe it's not an improvement. This boolean expressions are always harder than I want them to be :-) It's a shame JS doesn't have a built in [identity function](https://package.elm-lang.org/packages/elm/core/latest/Basics#identity) to use as the callback for `some`.

Reply:
I really like your suggestion and have refactored accordingly.
Much better like this:

`export const isEpicSuitable = ({`
`    shouldHideReaderRevenue,`
`    isMinuteArticle,`
`   isPaidContent,`
`}: {`
`    shouldHideReaderRevenue: boolean;`
`   isMinuteArticle: boolean;`
`    isPaidContent: boolean;`
`}): boolean => !(shouldHideReaderRevenue || isMinuteArticle || isPaidContent);`